### PR TITLE
Add new company - XP Inc 

### PR DESCRIPTION
**Pull request by anamanzandev**
If updating or adding a company, please include details here:  adding [XP Inc](https://www.xpi.com.br)

- [ ] This PR contains housekeeping only (URL edits, copy changes etc) - if check, delete other lines that don't apply.
- [ ] I am an employee of the company mentioned and confirm all included details are correct.
- [X] You know your alphabet - company is listed in alphabetical order in the README.md file.
- [X] The company directly hires employees. No bootcamps / freelance sites / etc
- [X] The company hires remote employees, or positions are available to remote workers and are clearly illustrated as such.
- [X] A [company profile](https://github.com/remoteintech/remote-jobs/blob/master/company-profiles/example.md) is included - This is __required__ for new additions.
- [X] __Remote status__ has details regarding how the culture includes remote employees, how the company integrated remote workers, etc
- [X] __Region__ details the geographic regions in which this company's employees can reside. For more details see the instructions in the [example company profile](/company-profiles/example.md#region).
- [X] __How to apply__ details the best approach for new applications, page on site where open position are listed, and any other help available for job hunters

This is a modified version of the [Contributing Guidelines](https://github.com/remoteintech/remote-jobs/blob/master/CONTRIBUTING.md).

This Pull Request should adhere to the repository's [Code of Conduct](https://github.com/remoteintech/remote-jobs/blob/master/CODE_OF_CONDUCT.md).

### CODE REVIEW

**Comment by auto-nylen** [comment]
:robot: :warning:

_Sorry, this bot is currently having issues. It's probably @nylen's fault, not yours. We'll look into it._

:robot: :warning:

**Comment by auto-nylen** [comment]
:robot: :warning:

_Sorry, this bot is currently having issues. It's probably @nylen's fault, not yours. We'll look into it._

:robot: :warning:

File: company-profiles/xp-inc.md [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+
+## Company size
+
+1001-5000
+
+## Remote status
+
+100% Remote
+
+## Region
```
Comment (dougaitken) [inline company-profiles/xp-inc.md:15]: Hey @anamanzandev - can you confirm if these are the only countries where employees can be hire? 

See this line for clarification - https://github.com/remoteintech/remote-jobs/blob/master/company-profiles/example.md#region

**Review by dougaitken — CHANGES_REQUESTED** [review]

File: company-profiles/xp-inc.md (hunk 1 above)
Comment (anamanzandev) [reply company-profiles/xp-inc.md:15]: Hi @dougaitken ,

According to their LinkedIn and website, these are the regions they have employees, and they will have some of them working at Portugal soon.

**Review by dougaitken — APPROVED** [review]

File: company-profiles/xp-inc.md (hunk 1 above)
Comment (dougaitken) [reply company-profiles/xp-inc.md:15]: Thanks @anamanzandev, the reason I ask is that the Region should be where people can be employed, not where they currently have employees.

**Review by dougaitken — APPROVED** [review]
The company profile can be updated if needed later

**Pull request merged**